libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -129,16 +104,10 @@ bool AC_PolyFence_loader::boundary_breached(const Vector2l& location, uint16_t n
     return Polygon_outside(location, &points[start_num], num_points-start_num);
 }
 
-bool AC_PolyFence_loader::boundary_breached(const Vector2f& location, uint16_t num_points, const Vector2f* points, bool contains_return_point) const
-{
-    // exit immediate if no points
-    if (points == nullptr) {
-        return false;
-    }
-
-    // start from 2nd point if boundary contains return point (as first point)
-    uint8_t start_num = contains_return_point ? 1 : 0;
-
-    // check location is within the fence
-    return Polygon_outside(location, &points[start_num], num_points-start_num);
-}
+// Necessary to avoid linker errors
```

Comment:
I don't like this comment very much, looks like this is a workaround, but this is how it works - if you define the templates in a cpp file you have to declare which specific instances of that template are compiled in.

Reply:
So should we remove this comment and push into master?  I'd like to get this into master if this is the only thing holding it up.

Reply:
Wow, totally missed the comment from Fransisco by accident. For what it's worth I just copied the comment from AP_Math on the definitions of the various poly functions we were calling that were already using templates. No strong opinion about having it, I was just mimicking other behavior rather then giving it new thought :)